exception.go

```diff
@@ -96,12 +96,17 @@ func loadExceptions() {
 	log.Info("completed exception refresh")
 }
 
+// isException returns true iff an ip is an exception in the active tree
```

Comment:
typo: `iff`

Reply:
Oh... i did that on purpose "if and only if"